### ISSUE TITLE
Can't query float values

### ISSUE THREAD

**Issue by Sharaddition** [supabase-community/supabase-py#192]
On querying database with gte, lte, gt, lt for float or np.float32 values. It's throwing Api error.

Error message-
> APIError: {'message': 'invalid input syntax for type real: ""4.6079545""', 'code': '22P02', 'details': None, 'hint': None}

The field I tried querying are float4 and float8 data types.

**Additional context**
On querying ID with numeric data type all functions work perfectly fine.

**Comment by anand2312** [comment]
I'm not able to reproduce this, what version of supabase-py and postgrest-py is being used here? If the version of postgrest-py is less than 0.10.1, I'd recommend you try upgrading that to latest and try again.

To see what version is being used you can run `pip show postgrest-py`

**Comment by Sharaddition** [comment]
I tried upgrading the `postgrest-py` still the problem persists.

My code-
`supabase: Client = create_client(supaURL, supaKEY)`
`supaTable = supabase.table("DemoTable").select("*").eq("price",4.60795).execute().data`

Table-
![table](https://user-images.githubusercontent.com/90986869/164063511-7c240f5b-9f46-4280-b687-79671cb07cbd.jpg)

Error log-

> ValidationError: 1 validation error for APIResponse
data
  You are passing an API error to the data field. (type=value_error)

>The above exception was the direct cause of the following exception:

>APIError                                  Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/postgrest_py/_sync/request_builder.py in execute(self)

>APIError: {'message': 'invalid input syntax for type real: ""4.60795""', 'code': '22P02', 'details': None, 'hint': None}

I forgot to mention I'm working on google colab can it be the problem? Btw all other function with timestamp and int8 are working fine.

**Comment by anand2312** [comment]
Again, can you tell what versions of supabase-py and postgrest-py are being used here? I cannot reproduce this but that might be because I am running the version that's on the master branch and the issue exists in an older version

**Comment by Sharaddition** [comment]
Hey @anand2312 these are the versions-

> Name: supabase
Version: 0.5.3

> Name: postgrest-py
Version: 0.10.2

**Comment by anand2312** [comment]
That's odd. This bug is known to be fixed in postgrest-py 0.10.1; and postgrest-py 0.10.2 is _incompatible_ with supabase-py 0.5.3 (it would raise import errors)
This makes me think you're somehow using the wrong Python. Maybe you have installed the libraries to a venv, but are running the files using the global Python installation, or something like that.

What happens if you just run
```py
import supabase
import postgrest_py

print(supabase.__version__)
print(postgrest_py.__version__)
``` ?

**Comment by Sharaddition** [comment]
These are the versions-

> 0.5.3
0.9.2

When I try to upgrade the postgrest_py to 0.10.1
`supabase 0.5.3 requires postgrest-py<0.10.0,>=0.9.1, but you have postgrest-py 0.10.1 which is incompatible.`

**Comment by anand2312** [comment]
Ah, looks like we can't do much about it then. A new release of supabase-py will be made sometime soon with these dependencies upgraded, so your best bet would be to wait for that now.
If you are only using the database features, you could just not use supabase-py and directly use postgrest-py as well.

**Comment by anand2312** [comment]
Fixed in release 0.5.5
